src/main/java/beam/physsim/cch/GraphReader.scala

```diff
@@ -0,0 +1,84 @@
+package beam.physsim.cch
+import java.io.File
+import java.nio.{ByteBuffer, ByteOrder}
+
+import com.google.common.io.Files
+import com.vividsolutions.jts.geom.Coordinate
+
+import scala.collection.mutable
+
+case class Vertex(id: Long, coordinate: Coordinate)
+case class RoutingFrameworkGraph(vertices: Seq[Vertex])
+
+trait RoutingFrameworkGraphReader {
+  def read(graph: File): RoutingFrameworkGraph
+}
+
+class RoutingFrameworkGraphReaderImpl extends RoutingFrameworkGraphReader {
```

Comment:
General idea: Is it possible to use something like [MessagePack](https://github.com/msgpack/msgpack-c)/[ProtoBuf](https://developers.google.com/protocol-buffers/docs/cpptutorial) for data exchange? This will allow you to make the changes easier and you don't need to think about Big or Little Indians.

Reply:
Sounds good, but in this case we need to change work with graph in routing framework

Reply:
As per the discussion this is not needed for now.